src/status_im/translations/ja.cljs

```diff
@@ -158,83 +177,204 @@
    :invited                               "招待済み"
    :removed                               "削除済み"
    :You                                   "あなた"
 
-   ;new-contact
+   ;;new-contact
    :add-new-contact                       "新規連絡先を追加"
-   :scan-qr                               "QRをスキャン"
+   :scan-qr                               "QRコードをスキャン"
    :name                                  "名前"
-   :address-explication                   "ここにアドレスについての説明や、どこで見つけられるのかを入力してください"
+   :address-explication                   "あなたの公開鍵は、Ethereum上にあなたのアドレスを生成するために使用され、一連の数字と文字です。 あなたのプロフィールで簡単に見つけることができます"
    :enter-valid-public-key                "有効な公開鍵を入力するかQRコードをスキャンしてください"
-   :contact-already-added                 "連絡先はすでに追加されています"
-   :can-not-add-yourself                  "自分自身を追加することはできません"
-   :unknown-address                       "不明なアドレス"
-
+   :contact-already-added                 "連絡先は既に追加されています"
+   :can-not-add-yourself                  "自分を追加することはできません"
+   :unknown-address                       "不明な住所"
 
-   ;login
+   ;;login
    :connect                               "接続"
    :address                               "アドレス"
    :password                              "パスワード"
    :sign-in-to-status                     "ステータスにサインイン"
    :sign-in                               "サインイン"
    :wrong-password                        "パスワードが間違っています"
+   :enter-password                        "パスワードを入力する"
 
-   ;recover
+   ;;recover
    :passphrase                            "パスフレーズ"
    :recover                               "復元"
-   :twelve-words-in-correct-order         "正しい順序で12ワード"
+   :twelve-words-in-correct-order         "正しい順序で12単語を並ぶ"
 
-   ;accounts
+   ;;accounts
    :recover-access                        "アクセスを復元"
    :create-new-account                    "新規アカウントを作成"
 
-   ;wallet-qr-code
+   ;;wallet-qr-code
    :done                                  "完了"
-   :main-wallet                           "メインウォレット"
 
-   ;validation
+   ;;validation
    :invalid-phone                         "無効な電話番号"
    :amount                                "金額"
-   ;transactions
+
+   ;;transactions
    :confirm                               "確認"
-   :transaction                           "トランザクション"
+   :transaction                           "取引"
+   :unsigned-transaction-expired          "署名されていない取引は期限切れです"
    :status                                "ステータス"
    :recipient                             "受信者"
    :to                                    "宛先"
    :from                                  "送信元"
    :data                                  "データ"
    :got-it                                "理解しました"
+   :block                                 "ブロック"
+   :hash                                  "ハッシュ"
+   :gas-limit                             "gas制限"
+   :gas-price                             "gas価格"
+   :gas-used                              "使用Gas"
+   :cost-fee                              "費用/料金"
+   :nonce                                 "現在時間"
```

Comment:
@andytudhope What does "nonce" mean here?

Reply:
Nonce is a technical word that refers to a unique number used to identify transactions: https://en.wikipedia.org/wiki/Cryptographic_nonce